pkg/shared/logging/log.go

```diff
@@ -33,6 +34,7 @@ func NewLogger() *zap.SugaredLogger {
 		config = zap.NewProductionConfig()
 	}
 	// Config customization goes here if any
+	config.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
```

Comment:
ODL follows `RFC3339xxx` any particular reason for `ISO8601`. either ways i am fine :)